jenkins-dcp.planx-pla.net/portal/gitops.json

```diff
@@ -290,56 +337,445 @@
     ],
     "arrangerConfig": {
```

Comment:
you might consider remving this guy, `arranger` has gone, and the portal config is already insanely long

Reply:
ok, lemme get rid of it. 👍🏼 Thanks!